examples/large_models/deepspeed/opt/requirements.txt

```diff
@@ -1,2 +1,2 @@
-transformers
-deepspeed
+transformers==4.28.1
```

Comment:
Is there a reason for pinning the version numbers? Can we do >= instead?

Reply:
The version combination of DS and transformer is not stable. Some combination does not work. So far this combination works for this example.